docs/reference/migration/migrate_7_0/aggregations.asciidoc

```diff
@@ -22,4 +22,5 @@ The object used to share aggregation state between the scripts in a Scripted Met
 Aggregation is now a variable called `state` available in the script context, rather than
 being provided via the `params` object as `params._agg`.
 
-The old `params._agg` variable is still available as well.
+The old `params._agg` variable is still available as well. This can be disabled by setting
```

Comment:
Since this will be backported to 6.x, with the intention of removing in 7.0, I don't think this migration note makes sense. The migration note would be that params._agg is gone, and would be in the followup which removes support in master only.

Reply:
Good catch, thanks. I overthought this and was going to remove it in my followup, but you're right; there's no need for this note here now that the target version is more solid.

Reply:
I'll revert the doc change such that this file is left as-is until the next PR.